### ISSUE TITLE
Incorrect Hohmann maneuver when not at pericenter?

### ISSUE THREAD

**Issue by astrojuanlu** [poliastro/poliastro#740]
🐞 **Problem**

I think there is a potential problem here:

https://github.com/poliastro/poliastro/blob/bc4c00887a6263601143366007edb6b7a711be61/src/poliastro/maneuver.py#L100-L101
https://github.com/poliastro/poliastro/blob/bc4c00887a6263601143366007edb6b7a711be61/src/poliastro/maneuver.py#L144

in that the time to get to the pericenter is not being taken into account for the maneuver, and therefore perhaps doing an `apply` on the initial orbit won't have the intended effect.

💡 **Possible solutions**

_Note I didn't check whether this is an actual bug or not yet_

Do not hardcode the initial maneuver to `0 * u.s` and use the time to get to the pericenter.

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

**Comment by jorgepiloto** [comment]
Interesting... The same happens in `Bielliptic` ones. Some possible solutions that came to my mind:

* We could raise `ValueError("Not in pericenter...")` so the user is forced to propagate before solving the maneuver.
* In case-solving internally time to pericenter, the set of impulses should be like:

```python
{propagation: till periapsis, second: first hohmann impulse, third: second hohmann impulse}
```

That propagation impulse always has `deltaV = 0 * u.km / u.s` and time may vary between [0 and T)

**Comment by Juanlu001** [comment]
I would rather do the latter instead of raising the error.

**Comment by Juanlu001** [comment]
Fixed in #744.

**Comment by astrojuanlu** [comment]
I think I forgot to close this. Doing it now.
